Source/Fuse.Controls.ScrollView/Triggers/WhileVisibleInScrollView.uno

```diff
@@ -6,6 +6,15 @@ using Fuse.Controls;
 namespace Fuse.Triggers
 {
 	/**
+		How the bounds of an element are treated.
+	*/
+	public enum WhileVisibleInScrollViewHow
+	{
+		Partial,
```

Comment:
Describe each enum value here with /** docs */. It need not be long but it's helpful for the API docs.